test/ex_integrate_test.exs

```diff
@@ -2,7 +2,25 @@ defmodule ExIntegrateTest do
   use ExUnit.Case
   doctest ExIntegrate
 
-  test "run steps" do
-    assert ExIntegrate.run_steps("test/ei.test.json") == :ok
+  @config_fixture_path "test/fixtures/ei.test.json"
+
+  describe "running the steps" do
+    test "success: returns :ok" do
+      assert :ok = ExIntegrate.run_steps(@config_fixture_path)
+    end
+
+    test "when file doesn't exist, raises error" do
+      assert_raise File.Error,
+        fn -> ExIntegrate.run_steps("nonexistant_file") end
```

Comment:
This makes me realize we should probably handle this somehow. Probably in the set up of a pipeline (in the API or later UI) we can let the user add a few mapping items like the repo URL, and the location of the build file so that it can be more flexible for a user of this.

Reply:
...If we require the user to add the file location as a workdir in the set up of a pipeline, then we could deal with it as a required data point

Reply:
> ...If we require the user to add the file location as a workdir in the set up of a pipeline, then we could deal with it as a required data point

Good idea! If we validate it at the entrypoint, we should be able to trust the path exists throughout the app and avoid needless error handling.

Reply:
@garrettmichaelgeorge maybe we'll revisit this one, and just keep it in a standard place in a repo for now